network/src/commonMain/kotlin/com/wire/kalium/network/api/KtorHttpClient.kt

```diff
@@ -2,26 +2,17 @@ package com.wire.kalium.network.api
 
 import com.wire.kalium.network.tools.HostProvider
 import com.wire.kalium.network.tools.KtxSerializer
-import io.ktor.client.HttpClient
-import io.ktor.client.call.receive
-import io.ktor.client.engine.HttpClientEngine
-import io.ktor.client.features.ClientRequestException
-import io.ktor.client.features.RedirectResponseException
-import io.ktor.client.features.ResponseException
-import io.ktor.client.features.ServerResponseException
-import io.ktor.client.features.auth.Auth
-import io.ktor.client.features.auth.providers.BearerTokens
-import io.ktor.client.features.auth.providers.bearer
-import io.ktor.client.features.defaultRequest
-import io.ktor.client.features.json.JsonFeature
-import io.ktor.client.features.json.serializer.KotlinxSerializer
-import io.ktor.client.features.websocket.WebSockets
-import io.ktor.client.request.header
-import io.ktor.client.request.host
-import io.ktor.client.statement.HttpResponse
-import io.ktor.http.ContentType
-import io.ktor.http.URLProtocol
-import io.ktor.util.toMap
+import io.ktor.client.*
+import io.ktor.client.engine.*
+import io.ktor.client.features.*
+import io.ktor.client.features.auth.*
+import io.ktor.client.features.auth.providers.*
+import io.ktor.client.features.json.*
+import io.ktor.client.features.json.serializer.*
+import io.ktor.client.features.websocket.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
```

Comment:
i think the .editorconfig file is not working for some reason 
make sure it's enabled in settings -> Code Style -> Enable EditorConfig support

Reply:
I think a different setting is msising in `EditorConfig`.

IntelliJ has these annoying settings by default:

![grafik](https://user-images.githubusercontent.com/9389043/144834552-c2214624-2e84-458f-a900-31120d2f478f.png)

When I've added the `.editorconfig`, IntelliJ didn't generate one that disabled them. I'll raise a PR with the missing `.editorconfig` property to fix this.